openapi.json

```diff
@@ -12,6 +12,12 @@
     "basic_auth": {
       "type": "basic"
     },
+    "oauth2": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "http://varnish:8080/oauth/authorize",
+      "tokenUrl": "http://varnish:8080/oauth/token"
```

Comment:
Are these internal urls, and if yes, is it working as suspected?

Reply:
These are internal yes. So for instance on my local environment it would be:
http://dapple-cms.docker/oauth/token

And the PR environment it is:
https://varnish.pr-1737.dpl-cms.dplplat01.dpl.reload.dk/oauth/token